pyproject.toml

```diff
@@ -16,13 +16,13 @@ cdislogging = "^1.0.0"
 cdiserrors =  "^0.1.2"
 doiclient = {git = "https://github.com/uc-cdis/doiclient", rev = "1.0.0"}
 dosclient = {git = "https://github.com/uc-cdis/dosclient", rev = "1.1.0"}
-gen3authz = "^0.4.0"
+gen3authz = "^1.0.4"
 hsclient = {git = "https://github.com/uc-cdis/hsclient", rev = "1.0.0"}
 indexclient = "^2.1.0"
 jsonschema = "^2.5.1"
 flask = "^1.1.1"
 psycopg2 = "^2.7"
-sqlalchemy = "^1.3.3"
+sqlalchemy = "~1.3.3"
```

Comment:
Could you remind me why we're changing this?

Reply:
It's being changed because of compatibility with sqlalchemy_utils! 1.4 does
not support sqlalchemy_utils yet